.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on: push
+
+jobs:
+  install-and-test:
+    name: ${{matrix.os}}
+    runs-on: ${{matrix.os}}
+    defaults:
+      run:
+        shell: bash -l {0}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, macos-10.15, windows-2019]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Create measelab conda environment
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          auto-activate-base: false
+          python-version: 3.8
+          activate-environment: measelab
+          environment-file: mease-env.yml
```

Comment:
for the time being, I would point to the master versions of spikeextractors and neo until they are released ;)

Reply:
thanks @alejoe91 - I did this (also had to do the same for `nwb-conversion-tools` and add a couple of dependencies) and now it seems to be working